Modules/timemodule.c

```diff
@@ -2066,14 +2070,24 @@ pysleep(_PyTime_t secs)
         return -1;
     }
     deadline = monotonic + secs;
+#ifdef HAVE_CLOCK_NANOSLEEP
+    if (_PyTime_AsTimespec(deadline, &timeout_abs) < 0)
```

Comment:
Please add { ... } (PEP 7). Please also add them to the existing _PyTime_AsTimeval() call.